### PR TITLE
[Documentation][Cookbook] How to disable default shop, admin or API of Sylius?

### PR DESCRIPTION
**Pull request by TheMadeleine**
| Q               | A
| --------------- | ---
| Bug fix?        | no |
| New docs?    | yes |
| Related tickets |  |
| License         | MIT |

### CODE REVIEW

File: docs/cookbook/disabling-shop-admin-api.rst [hunk 1]
```diff
@@ -0,0 +1,193 @@
...
+        - { resource: "@SyliusShopBundle/Resources/config/app/config.yml" }
+
+**3. Remove SyliusShopBundle routing configuration from ``app/config/routing.yml``**
+
+.. code-block:: yaml
+
+    sylius_shop:
+        resource: "@SyliusShopBundle/Resources/config/routing.yml"
+
+**4. Remove security configuration form ``app/config/security.yml``**
```
Comment (GSadee) [inline docs/cookbook/disabling-shop-admin-api.rst]: from

File: docs/cookbook/disabling-shop-admin-api.rst [hunk 2]
```diff
@@ -0,0 +1,193 @@
...
+
+The part that has to be removed form this file is shown below:
```
Comment (GSadee) [inline docs/cookbook/disabling-shop-admin-api.rst]: from

Comment (NeverResponse) [inline docs/cookbook/disabling-shop-admin-api.rst]: `from` here, and two lines up

File: docs/cookbook/disabling-shop-admin-api.rst [hunk 3]
```diff
@@ -0,0 +1,193 @@
...
+        - { resource: "@SyliusAdminBundle/Resources/config/app/config.yml" }
+
+**3. Remove SyliusAdminBundle routing configuration from ``app/config/routing.yml``**
+
+.. code-block:: yaml
+
+    sylius_shop:
+        resource: "@SyliusAdminBundle/Resources/config/routing.yml"
+
+**4. Remove security configuration form ``app/config/security.yml``**
```
Comment (GSadee) [inline docs/cookbook/disabling-shop-admin-api.rst]: from

File: docs/cookbook/disabling-shop-admin-api.rst [hunk 4]
```diff
@@ -0,0 +1,193 @@
...
+
+The part that has to be removed form this file is shown below:
```
Comment (GSadee) [inline docs/cookbook/disabling-shop-admin-api.rst]: from

File: docs/cookbook/disabling-shop-admin-api.rst [hunk 5]
```diff
@@ -0,0 +1,193 @@
...
+        - { resource: "@SyliusApiBundle/Resources/config/app/config.yml" }
+
+**3. Remove SyliusApiBundle routing configuration from ``app/config/routing.yml``**
+
+.. code-block:: yaml
+
+    sylius_shop:
+        resource: "@SyliusApiBundle/Resources/config/routing.yml"
+
+**4. Remove security configuration form ``app/config/security.yml``**
```
Comment (GSadee) [inline docs/cookbook/disabling-shop-admin-api.rst]: from

File: docs/cookbook/disabling-shop-admin-api.rst [hunk 6]
```diff
@@ -0,0 +1,193 @@
...
+
+The part that has to be removed form this file is shown below:
```
Comment (GSadee) [inline docs/cookbook/disabling-shop-admin-api.rst]: from

File: docs/cookbook/disabling-shop-admin-api.rst (hunk 2 above)
Comment (TheMadeleine) [inline docs/cookbook/disabling-shop-admin-api.rst]: That's my favourite spelling mistake <3 :tada:

File: docs/cookbook/disabling-shop-admin-api.rst [hunk 7]
```diff
@@ -0,0 +1,193 @@
...
+**4. Remove security configuration from ``app/config/security.yml``**
+
+The part that has to be removed from this file is shown below:
+
+.. code-block:: yaml
+
+    # security:
+        # firewalls:
+            shop:
+            switch_user: { role: ROLE_ALLOWED_TO_SWITCH }
```
Comment (NeverResponse) [inline docs/cookbook/disabling-shop-admin-api.rst]: wrong indentation

File: docs/cookbook/disabling-shop-admin-api.rst [hunk 8]
```diff
@@ -0,0 +1,193 @@
...
+        # format_listener:
+            # rules:
+                - { path: '^/api', priorities: ['json', 'xml'], fallback_format: json, prefer_extension: true }
+
+**Done!** There is no API in Sylius now, just admin and shop.
+
+Learn more
+----------
+
+* :ref:`Architecture: Division into Shop, Admin and API <division-into-shop-admin-api>`
```
Comment (NeverResponse) [inline docs/cookbook/disabling-shop-admin-api.rst]: Missing core

**Comment by pjedrzejewski** [comment]
Thanks Magda, nice work! ;)

File: docs/cookbook/disabling-shop-admin-api.rst [hunk 9]
```diff
@@ -0,0 +1,223 @@
...
+    }
+
+**2. Remove SyliusApiBundle's config import from ``app/config/config.yml``**
+
+Here you've got the line that should disappear from imports:
+
+.. code-block:: yaml
+
+    imports:
+    #    - { resource: "@SyliusApiBundle/Resources/config/app/config.yml" } # remove or leave this line commented
```
Comment (pamil) [inline docs/cookbook/disabling-shop-admin-api.rst]: Same as for `rules` below.

File: docs/cookbook/disabling-shop-admin-api.rst [hunk 10]
```diff
@@ -0,0 +1,223 @@
...
+    }
+
+**2. Remove SyliusAdminBundle's config import from ``app/config/config.yml``**
+
+Here you've got the line that should disappear from imports:
+
+.. code-block:: yaml
+
+    imports:
+    #    - { resource: "@SyliusAdminBundle/Resources/config/app/config.yml" } # remove or leave this line commented
```
Comment (pamil) [inline docs/cookbook/disabling-shop-admin-api.rst]: Same as for `rules` below.

File: docs/cookbook/disabling-shop-admin-api.rst [hunk 11]
```diff
@@ -0,0 +1,223 @@
...
+    #        resource: "@SyliusAdminBundle/Resources/config/routing.yml"
+
+**4. Remove security configuration from ``app/config/security.yml``**
+
+The part that has to be removed from this file is shown below:
+
+.. code-block:: yaml
+
+    security:
+        firewalls:
```
Comment (pamil) [inline docs/cookbook/disabling-shop-admin-api.rst]: Same as for `rules` below.

File: docs/cookbook/disabling-shop-admin-api.rst [hunk 12]
```diff
@@ -0,0 +1,223 @@
...
+    # Delete or leave this part commented
```
Comment (pamil) [inline docs/cookbook/disabling-shop-admin-api.rst]: Same as for `rules` below.

File: docs/cookbook/disabling-shop-admin-api.rst [hunk 13]
```diff
@@ -0,0 +1,223 @@
...
+    }
+
+**2. Remove SyliusShopBundle's config import from ``app/config/config.yml``**
+
+Here you've got the line that should disappear from imports:
+
+.. code-block:: yaml
+
+    imports:
+    #    - { resource: "@SyliusShopBundle/Resources/config/app/config.yml" } # remove or leave this line commented
```
Comment (pamil) [inline docs/cookbook/disabling-shop-admin-api.rst]: Same as for `rules` below.

File: docs/cookbook/disabling-shop-admin-api.rst [hunk 14]
```diff
@@ -0,0 +1,223 @@
...
+    #        anonymous:  true
+
+**5. Remove fos_rest config from ``app/config/config.yml``**
+
+.. code-block:: yaml
+
+    fos_rest:
+        format_listener:
+            rules:
+            #    - { path: '^/api', priorities: ['json', 'xml'], fallback_format: json, prefer_extension: true } # remove or leave this line commented
```
Comment (TheMadeleine) [inline docs/cookbook/disabling-shop-admin-api.rst]: It's not the only child :)

**Pull request merged**